### PR TITLE
Lets add coverage for the things we broke

### PR DESCRIPTION
**Pull request by jaym**
cc @btm @lamont-granquist @chef/client-core

### CODE REVIEW

**Comment by btm** [comment]
:clap: ...\*waits patiently for travis\*

**Comment by jdmundrawala** [comment]
@btm just to be clear, you are patiently waiting for travis to fail for 2 of the 4 new cases that have been added and not horribly blow up

File: kitchen-tests/cookbooks/installthings/attributes/default.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+case node['platform_family']
+when "debian"
+  default['installthings']['push_client_url'] =
+    'https://opscode-private-chef.s3.amazonaws.com/ubuntu/12.04/x86_64/opscode-push-jobs-client_1.1.5-1_amd64.deb'
```
Comment (lamont-granquist) [inline kitchen-tests/cookbooks/installthings/attributes/default.rb]: can we find some smaller debs/rpms to install?  also the network dependency here will break at some point in time, if we can get a tiny rpm + deb checked into the cookbook they could just be installed locally.

File: .travis.yml [hunk 2]
```diff
@@ -44,9 +44,9 @@ matrix:
     - cd kitchen-tests
     script:
 # FIXME: we should fix centos-6 against AWS and then enable it here
-    - if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]; then bundle exec kitchen test ubuntu; fi
+    - if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]; then bundle exec kitchen test ubuntu installthings --concurrency 8; fi
```
Comment (lamont-granquist) [inline .travis.yml]: this is just for testing and at some point installthings is added to the default run_list?

Comment (jdmundrawala) [inline .travis.yml]: this just prevents the webapp stuff from running on centos as those tests are broken

File: kitchen-tests/cookbooks/installthings/attributes/default.rb (hunk 1 above)
Comment (jdmundrawala) [inline kitchen-tests/cookbooks/installthings/attributes/default.rb]: got any ideas here? i was considering making a tiny omnibus package with something like a text file

Comment (danielsdeleo) [inline kitchen-tests/cookbooks/installthings/attributes/default.rb]: There's something like this either in the repo already or in the history.

Comment (lamont-granquist) [inline kitchen-tests/cookbooks/installthings/attributes/default.rb]: like https://github.com/chef/chef/blob/master/spec/data/apt/chef-integration-test_1.0-1_amd64.deb

could use fpm to create an equivalently trivial rpm to install.

Comment (jdmundrawala) [inline kitchen-tests/cookbooks/installthings/attributes/default.rb]: that sounds easier

File: kitchen-tests/cookbooks/installthings/.kitchen.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (tyler-ball) [inline kitchen-tests/cookbooks/installthings/.kitchen.yml]: Does this cookbook need its own `.kitchen.yml`?

**Comment by tyler-ball** [comment]
:+1:

**Comment by jdmundrawala** [comment]
This is blocked until I can figure out why kitchen is hanging

**Comment by tyler-ball** [comment]
@jdmundrawala can you try this again?  I updated kitchen-ec2 this morning

**Comment by lamont-granquist** [comment]
should we close this if you don't have time to get to it?

**Comment by jaym** [comment]
we're testing in other ways now

**Pull request closed without merging**